bitbots_localization/src/parameters.yml

```diff
@@ -0,0 +1,241 @@
+      validation:
+        bounds<>: [0.0, 1.0]
+    min_motion_angular:
+      type: double
+      description: "Minimum angular motion which is considered to be a movement. This is relevant if we want to deactivate the filter if no movement is detected"
+      validation:
+        bounds<>: [0.0, 1.0]
+    filter_only_with_motion:
+      type: bool
+      description: "If true, the filter is only active if a movement is detected"
+  ros:
+    line_pointcloud_topic:
+      type: string
+      description: "Topic for the line pointcloud input messages"
+      read_only: true
+    goal_topic:
+      type: string
+      description: "Topic for the goal input messages"
+      read_only: true
+    fieldboundary_topic:
+      type: string
+      description: "Topic for the field boundary input messages"
+      read_only: true
+    particle_publishing_topic:
+      type: string
+      description: "Topic for the particle publishing messages"
+      read_only: true
+    debug_visualization:
+      type: bool
+      description: "Activate debug visualization"
+    odom_frame:
+      type: string
+      description: "The odometry frame name"
+      read_only: true
+    base_footprint_frame:
+      type: string
+      description: "The base footprint frame name"
+      read_only: true
+    map_frame:
+      type: string
+      description: "The map frame name"
+      read_only: true
+    publishing_frame:
+      type: string
+      description: "The frame which is used for publishing the pose estimate"
+      read_only: true
+  particle_filter:
+    particle_number:
+      type: int
+      description: "Number of particles"
+      read_only: true
+      validation:
+        bounds<>: [1, 10000]
+    rate:
+      type: int
+      description: "Frequency at which the filter is updated"
+      read_only: true
+      validation:
+        bounds<>: [1, 200]
+    resampling_interval:
+      type: int
+      description: "Number of steps after which resampling is performed"
+      validation:
+        gt<>: [0]
+    diffusion:
+      x_std_dev:
+        type: double
+        description: "Standard deviation for the diffusion in x direction"
+        validation:
+          gt_eq<>: [0.0]
+      y_std_dev:
+        type: double
+        description: "Standard deviation for the diffusion in y direction"
+        validation:
+          gt_eq<>: [0.0]
+      t_std_dev:
+        type: double
+        description: "Standard deviation for the diffusion in theta (rotation) direction"
+        validation:
+          gt_eq<>: [0.0]
+      multiplier:
+        type: double
+        description: "Scaling multiplier for the diffusion (hacky way to change the diffusion strength)"
+        validation:
+          gt_eq<>: [0.0]
+      starting_multiplier:
+        type: double
+        description: "Starting diffusion multiplier value. This can be used to increase the diffusion at after initialization, which can help with initial exploration and convergence"
```

Comment:
Which one? `at initialization` or `after initialization`?

Reply:
It should be shortly after I will change that